Sign_Up_Demo/Controllers/BaseViewController.swift

```diff
@@ -0,0 +1,18 @@
+import UIKit
+
+class BaseViewController: UIViewController, UIGestureRecognizerDelegate {
+    
+    // MARK: - Life Cycle Methods
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        disableDarkMode()
+        navigationController?.interactivePopGestureRecognizer?.delegate = self
```

Comment:
Why it is in base controller ?

Reply:
I thought this functionality should be available for all the controllers, but it is not so.